templates/admin-header.php

```diff
@@ -134,7 +134,7 @@
                             </div>
                             <?php esc_html_e( 'Basic & Fundamental', 'dokan-lite' ); ?>
                         </a>
-                        <a href="https://wedevs.com/account/dokan-feature-requests/" target="_blank">
+                        <a href="https://pluginfeedback.dokan.co/" target="_blank">
```

Comment:
- Pls update the url to `https://pluginfeedback.dokan.co/b/feature-requests`
- Pls also add utm query params.

CC: @getdokan/product-plugin-team